util/dvsim/dvsim.py

```diff
@@ -114,6 +114,30 @@ def resolve_max_parallel(arg):
     return 16
 
 
+def read_verbosity(arg):
```

Comment:
The original motivation for `dvsim` was to remain agnostic to the underlying technology used for running the simulations (UVM DV, Verilator or whatever). The 'generic' tool switches {n, l, m, h, d} would then transform to simulation-technology specific verbosity levels supplied in the HJson. I prefer to keep the codebase free from any assumptions (such as simulations are -only- run with UVM enabled), and instead maintain things like these in the 'user' data (sim cfg HJson files). I accept the fact that there is a lack of documentation explaining the design choices. 

I also agree that the mysterious error messages that get thrown due to totally unrelated setup issues should be fixed, but maybe in some other way.

Reply:
Would you accept a patch that changed the magic names in the hjson files from `l`, `m` etc. to `verbosity_to_uvm_l`, `verbosity_to_uvm_m` etc?